src/Database/DatabaseServiceProviderEncrypt.php

```diff
@@ -15,6 +15,11 @@ class DatabaseServiceProviderEncrypt extends DatabaseServiceProvider
      */
     protected function registerConnectionServices()
     {
+        $this->mergeConfigFrom(
+            __DIR__ . '/../config/aesEncrypt.php',
```

Comment:
I thought the convention was lowercase with either hyphens or underscores?

Reply:
As discussed opting for this naming scheme for compatibility reasons with original package.